yarn.lock

```diff
@@ -0,0 +1,2780 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Please remove. This project does not use yarn.

Reply:
My bad. Used to adding a lockfile on projects since I mostly work with applications instead of libraries.